src/sentry/tasks/post_process.py

```diff
@@ -411,6 +411,17 @@ def fetch_buffered_group_stats(group):
     group.times_seen_pending = result["times_seen"]
 
 
+def fetch_buffered_group_last_seen(event, group):
+    """
+    Fetches buffered updates to `last_seen` for this group.
+    """
+    from sentry import buffer
+    from sentry.models import Group
+
+    result = buffer.get(Group, ["last_seen"], {"pk": group.id})
```

Comment:
Looks like when I wrote `buffer.get` I only handled integer columns https://github.com/getsentry/sentry/blob/b44945330fbbf535c79283e14cf2803d65417bd3/src/sentry/buffer/redis.py#L179-L196

This method will need to be updated to also handle datetimes